src/main/java/org/vanilladb/core/storage/tx/concurrency/ConcurrencyMgr.java

```diff
@@ -178,4 +179,8 @@ public void lockRecordFileHeader(BlockId blk) {
 	public void releaseRecordFileHeader(BlockId blk) {
 		lockTbl.release(blk, txNum, LockTable.X_LOCK);
 	}
+	
+	public ReentrantLock getLockForFileHeader(BlockId blk) {
+		return null;

```

Comment:
We better give it a global latch here to ensure that VanillaCore still works alone.

Reply:
Got it